journals/apps/journals/journal_page_helper.py

```diff
@@ -61,6 +61,24 @@ def get_serializer(self, request):
 
         return serializer
 
+    def serve(self, request, *args, **kwargs):
+        """
+        Override serve to redirect to frontend app
+        This gets called whenever a page is requested to be viewed live
+        from the Wagtail admin
+        """
+        if not settings.FRONTEND_ENABLED:
+            return super(JournalPageMixin, self).serve(request, args, kwargs)
+
+        response = redirect(
+            '{frontend_url}/{page_path}'.format(
```

Comment:
NIT: We might want to use urljoin (urllib.parse.urljoin) in case frontend_url has a trailing slash.

Reply:
fixed